docs/native/src/overview.rst

```diff
@@ -1,5 +1,76 @@
 
 ..  _intro:
 
-|stub-icon| Integrating with Office for iOS
-===========================================
+Integrating with Office for iOS
```

Comment:
FWIW, I am looking to remove this entire section in the Office Online docs. The info it contains would fit better in various specific sections or the main page.